packages/react-native-codegen/scripts/oss/build.sh

```diff
@@ -32,12 +32,24 @@ if [[ ${FBSOURCE_ENV:-0} -eq 1 ]]; then
   "$YARN_BINARY" run build
 
   popd >/dev/null
+
 else
   # Run yarn install in a separate tmp dir to avoid conflict with the rest of the repo.
   # Note: OSS-only.
   TMP_DIR=$(mktemp -d)
 
-  cp -R "$CODEGEN_DIR/." "$TMP_DIR"
+  # On Windows this script gets run by a seprate Git Bash instance, which cannot perform the copy
+  # due to file locks created by the host process. Need to exclude .lock files while copying.
+  # Using in-memory tar operation because piping `find` and `grep` doesn't preserve folder structure
+  # during recursive copying, and `rsync` is not installed by default in Git Bash.
+  # As an added benefit, blob copy is faster.
+  if [ "$OSTYPE" = "msys" ]; then
```

Comment:
Potentially dumb question (I'm no Windows person): should this also check against `win*` and `cygwin`?

Reply:
I've only tested this with GitBash, but it should also work with cygwin - I'll add it.